src/i18n/es/index.js

```diff
@@ -0,0 +1,156 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  en: {
+    message: {
+      ...potato,
+      help: {
+        label: 'Abrir tutorial'
+      },
+      print: { ...print },
+      tester: { ...tester },
+      keyboard: {
+        label: 'teclado'
+      },
+      layout: {
+        label: 'plano'
```

Comment:
In this context I think it would be more often used the anglicism `layout` or the word `distribución` than the literal translation `plano`. Eg. ["layout del teclado"](https://www.google.com/search?hl=es&biw=1366&bih=622&tbm=isch&sxsrf=ACYBGNTSSoA2iM-D4jn6GrLhRi83ty0mkA%3A1570210827153&sa=1&ei=C4SXXcn-CJKBhbIPyp-C4Aw&q=%22layout+del+teclado%22&oq=%22layout+del+teclado%22&gs_l=img.3...36620.37427..37726...0.0..0.147.633.4j2......0....1..gws-wiz-img.......0i7i30.qtkfZX1MoU0&ved=0ahUKEwiJsMyUk4PlAhWSQEEAHcqPAMwQ4dUDCAY&uact=5) on google images vs ["plano del teclado"](https://www.google.com/search?hl=es&biw=1366&bih=622&tbm=isch&sxsrf=ACYBGNRC_Ss4ptdjIT5Z8__D0B2s7WhWHw%3A1570210768956&sa=1&ei=0IOXXer8OcmR8gLB24qYDQ&q=%22plano+del+teclado%22&oq=%22plano+del+teclado%22&gs_l=img.3...55542.55898..56144...0.0..1.167.562.2j3......0....1..gws-wiz-img.......0i7i30.0XR1L53K4wQ&ved=0ahUKEwjqpOz4koPlAhXJiFwKHcGtAtMQ4dUDCAY&uact=5) or even ["distribución del teclado"](https://www.google.com/search?hl=es&biw=1366&bih=622&tbm=isch&sxsrf=ACYBGNT4hwB7bvKqNpq4qF6ZAjcJ-FH4pA%3A1570210866610&sa=1&ei=MoSXXenhJNmD8gK_lLmIBQ&q=%22distribuci%C3%B3n+del+teclado%22&oq=%22distribuci%C3%B3n+del+teclado%22&gs_l=img.3..0l3j0i30l2j0i5i30.93238.94646..94864...0.0..0.108.1109.11j2......0....1..gws-wiz-img.......0i7i30j0i7i5i30j0i8i7i30.wEcAHgHIf9k&ved=0ahUKEwjpwrSnk4PlAhXZgVwKHT9KDlEQ4dUDCAY&uact=5)